### PR TITLE
Use "general" verb form in docstrings

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR aims to use more consistently the "general" verb form in the docstrings.

Fixes: https://github.com/GenericMappingTools/pygmt/pull/2345#discussion_r1110977228
Related to GMT upstream documentation: https://github.com/GenericMappingTools/gmt/commit/a57fd6c2f5b16c7fcaaaaa4b806233d19f0cd75b, https://github.com/GenericMappingTools/gmt/commit/8256073836925692451f5ece92e2cb81ec7fad26

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/src/binstats.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def binstats(data, **kwargs):
         Normalize the resulting grid values by the area represented by the
         search *radius* [no normalization].
     search_radius : float or str
-        Sets the *search_radius* that determines which data points are
+        Set the *search_radius* that determines which data points are
         considered close to a node. Append the distance unit.
         Not compatible with ``tiling``.
```
Comment (yvonnefroehlich) [inline pygmt/src/binstats.py:89]: At the moment ``tiling`` is not added as an alias  for **T**  (https://docs.generic-mapping-tools.org/latest/gmtbinstats.html#t).

File: pygmt/src/image.py [hunk 2]
```diff
@@ -43,11 +43,11 @@ def image(self, imagefile, **kwargs):
...
         **+w**\ [**-**]\ *width*\ [/*height*]\ [**+j**\ *justify*]\
         [**+n**\ *nx*\ [/*ny*] ]\ [**+o**\ *dx*\ [/*dy*]].
-        Sets reference point on the map for the image.
+        Set reference point on the map for the image.
     box : bool or str
         [**+c**\ *clearances*][**+g**\ *fill*][**+i**\ [[*gap*/]\ *pen*]]\
         [**+p**\ [*pen*]][**+r**\ [*radius*]][**+s**\ [[*dx*/*dy*/][*shade*]]].
-        Without further arguments, draws a rectangular border around the image
+        If set to ``True``, draws a rectangular border around the image
         using :gmt-term:`MAP_FRAME_PEN`.
```
Comment (yvonnefroehlich) [inline pygmt/src/image.py:51]: Not sure, if we should change "draws" to "draw" here?

**Comment by yvonnefroehlich** [comment]
No guarantee I found all occurrences... Thanks for reviewing this PR.

@GenericMappingTools/pygmt-maintainers should there be a separate PR for the other docstring changes in this PR or is leaving these changes in this PR fine?

File: pygmt/src/binstats.py (hunk 1 above)
Comment (yvonnefroehlich) [reply pygmt/src/binstats.py:89]: I submitted a PR to add this alias to the documentation of `pygmt.binstats`, please see PR #2409.

File: pygmt/src/image.py (hunk 2 above)
Comment (seisman) [reply pygmt/src/image.py:51]: I think it should be "draw".

**Review by seisman — APPROVED** [review]

File: pygmt/src/image.py (hunk 2 above)
Comment (yvonnefroehlich) [reply pygmt/src/image.py]: Thanks! This is changed in commit https://github.com/GenericMappingTools/pygmt/pull/2376/commits/3234d7679ff1777c0c0826dff31e5a4931c6d580.

File: pygmt/src/meca.py [hunk 3]
```diff
@@ -231,8 +231,9 @@ def meca(
         determined by the z-value (i.e., event depth or the third column for
         an input file).
     no_clip : bool
-        Does NOT skip symbols that fall outside frame boundary specified by
-        ``region`` [Default is False, i.e. plot symbols inside map frame only].
+        Do NOT skip symbols that fall outside frame boundary specified by
```
Comment (michaelgrund) [inline pygmt/src/meca.py:234]:
```suggestion
        Do NOT skip symbols that fall outside the frame boundary specified by
```

File: pygmt/src/meca.py [hunk 4]
```diff
@@ -231,8 +231,9 @@ def meca(
...
+        ``region`` [Default is ``False``, i.e., plot symbols inside map
```
Comment (michaelgrund) [inline pygmt/src/meca.py:235]:
```suggestion
        ``region`` [Default is ``False``, i.e., plot symbols inside the map
```

**Review by michaelgrund — APPROVED** [review]

File: pygmt/src/meca.py (hunk 3 above)
Comment (yvonnefroehlich) [reply pygmt/src/meca.py]: @michaelgrund in commit https://github.com/GenericMappingTools/pygmt/pull/2376/commits/2f94dd5d377d1731b87d3a5830d97fb6742a06b3 I _tried_ to make the docstrings for `no_clip` more consistent across the single methods.

**Pull request merged**